DividingStruct.cpp

```diff
@@ -0,0 +1,27 @@
+#include "DividingStruct.h"
+
+DividingStruct::DividingStruct(const SummingStruct& s)
+        : summing(s), multiplier(s)
+{
+}
```

Comment:
This coding style should be improved.  
One or more of the following coding standard guidelines is not being followed and revision is needed.
- Your function arguments should be on the same line as the opening `(`
- multiple function arguments should be aligned with that opening `(`
- the `:` should be placed after the closing `)`
- member variables being initialized in the Member Initializer List should be aligned with the left edge of the text editor
- [ ] revise